src/Benchmark/BenchEngine.cs

```diff
@@ -16,21 +16,19 @@ public BenchEngine(IObjectToObjectMapper mapper, string mode)
 
 		public void Start()
 		{
-			var timer = Stopwatch.StartNew();
-
 			_mapper.Initialize();
 			_mapper.Map();
 
-			timer.Start();
+            var timer = Stopwatch.StartNew();

```

Comment:
@lbargaoanu Yea so ` Stopwatch.StartNew()` starts counting immediately.
That's why I was seeing a big slowdown in performance looking at the numbers.